spec/mailers/notifier_spec.rb

```diff
@@ -213,14 +213,14 @@
     end
 
     it "SUBJECT: has a snippet of the post contents" do
-      expect(@mail.subject).to eq(@cnv.subject)
+      expect(@mail.subject).to eq(I18n.translate 'notifier.private_message.subject')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.